File: setup/default.rs
fn main() {
    let input = include_str!("./input.txt");

    let output = solve(input);
    println!("Part 1 answer: {}", output);
}

fn solve(input: &str) -> usize {
    $output
}

#[cfg(test)]
mod tests {
    use super::*;
    use indoc::indoc;

    #[test]
    fn example_part_1() {
        let input = indoc! {"
$input
        "};
        let result = solve(input);
        assert_eq!(result, $output);
    }
}

File: setup/default.py
from textwrap import dedent


def solve(input: str) -> int:
    return $output


def test_solve():
    input = dedent("""
$input
    """)
    assert solve(input) == $output


if __name__ == '__main__':
    with open('input.txt') as f:
        input = f.read()

    ans = solve(input)
    print(ans)
